python/ray/tune/trial.py

```diff
@@ -155,18 +146,12 @@ def __init__(self,
         self.last_update_time = -float("inf")
         self.checkpoint_freq = checkpoint_freq
         self.checkpoint_at_end = checkpoint_at_end
+        self.sync_on_checkpoint = sync_on_checkpoint
+        newest_checkpoint = Checkpoint(Checkpoint.DISK, restore_path)
+        self.checkpoint_manager = CheckpointManager(keep_checkpoints_num,
+                                                    checkpoint_score_attr)
+        self.checkpoint_manager.newest_checkpoint = newest_checkpoint
```

Comment:
should this go through the `on_checkpoint` codepath?

Reply:
idts, the checkpoint doesnt have a score so we cant put it in the PQ